tests/test_mcp_oauth.py

```
@@ -0,0 +1,1515 @@
+                    self.send_header("Content-Type", "text/html")
+                    self.end_headers()
+                    self.wfile.write(b"<h1>Authenticated! You can close this tab.</h1>")
+                else:
+                    self.send_response(400)
+                    self.end_headers()
+                    error = params.get("error", ["unknown"])[0]
+                    desc = params.get("error_description", [""])[0]
+                    auth_code_result["error"] = f"{error}: {desc}"
+                    self.wfile.write(f"<h1>Error: {error} - {desc}</h1>".encode())
+
+            def log_message(self, format, *args):
```

Ruff (0.15.9)

[error] Function argument `format` is shadowing a Python builtin

(A002)